strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/artifact/coordinates/PypiArtifactCoordinatesTest.java

```diff
@@ -95,7 +95,7 @@ public PypiArtifactCoordinatesTest()
      * Tests the constructor of PypiArtifactCoordinates with real Python package names.
      * Simply add more pre-parsed filenames to parsedPackages array to test additional filenames.
      */
-    @Test
+    @RepeatedTest(10)
     public void testManualCreateArtifact()
     {
         for (ArrayList<String> coords : parsedPackages)
```

Comment:
@Firstmol I mean that you should use inject `RepetitionInfo` as test parameter and then use `parsedPackages.get(RepetitionInfo.getCurrentRepetition())` instead of `foreach`

Reply:
oh sorry, ok

Reply:
fixed